src/main/scala/com/fulcrumgenomics/umi/CopyUmiFromReadName.scala

```diff
@@ -42,9 +42,7 @@ import com.fulcrumgenomics.util.{Io, ProgressLogger}
 class CopyUmiFromReadName
 ( @arg(flag='i', doc="The input BAM file") input: PathToBam,
   @arg(flag='o', doc="The output BAM file") output: PathToBam,
-  @arg(doc="Remove the UMI from the read name") removeUmi: Boolean = false,
-  @arg(doc="Replaces any occurrences of this delimiter found in the UMI with a dash ('-') as per the SAM specification")
-  umiDelimiter: Option[Char] = None
```

Comment:
The usage needs to be updated to say that UMIs can be either dash or plus delimited.